Cargo.lock

```diff
@@ -9005,12 +8895,12 @@ dependencies = [
  "sp-crypto-hashing",
  "sp-io",
  "sp-runtime",
- "sp-std 14.0.0",
+ "sp-std",
 ]
 
 [[package]]
 name = "pallet-asset-conversion"
-version = "10.0.0"
+version = "11.0.0"
```

Comment:
It seems to bump all crates?  
I think it should only bump the simple-mermaid crate.

Reply:
I think it just fixes Cargo.lock, if you check `pallet-asset-conversion` in this branch:
```
[package]
name = "pallet-asset-conversion"
version = "11.0.0"
```

Reply:
@Morganamilo do you know why the lock file is out of sync with this branch?